src/test/shell/bazel/platforms_test.sh

```diff
@@ -321,7 +321,7 @@ EOF
   bazel build --extra_execution_platforms=":my_platform" :a --execution_log_json_file out.txt || fail "Build failed"
   grep "platform_key" out.txt || fail "Did not find the platform key"
   grep "default_value" out.txt || fail "Did not find the default value"
-  grep "test_value" out.txt && fail "Used the test-action value when not testing"
+  (! grep "test_value" out.txt) || fail "Used the test-action value when not testing"
```

Comment:
I didn't know this syntax existed. I see the benefits, but it's very different than all the other tests, so can you please change this back to the previous style, and use the same style in your new tests? I think consistency is better in this case.

Reply:
Done. I couldn't figure out why, but one of the tests failed with that style (and I missed the test case with expected failure). I ended up removing it, as there is a positive test anyway.